src/examples/button/button.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"machine"
+	"runtime"
+)
+
+func main() {
+	led := machine.GPIO{machine.LED}
+	led.Configure(machine.GPIOConfig{Mode: machine.GPIO_OUTPUT})
+
+	button := machine.GPIO{8}
+	button.Configure(machine.GPIOConfig{Mode: machine.GPIO_INPUT})
+
+	led.High()
```

Comment:
This is not really necessary: the state is set to high or low immediately below.